gf-webapp/src/Index.scss

```diff
@@ -66,4 +66,53 @@ $input-padding: 10px;
 $input-font-size: 18px;
 $input-font-family: Montserrat Regular;
 $input-border: 2px solid $color-primary;
-$input-border-focus: 2px solid $color-secondary;
\ No newline at end of file
+$input-border-focus: 2px solid $color-secondary;
+
+//room titles styles
+$room-title-font-family: Montserrat Regular;
+
+//room h1 titles styles
+$room-h1-display: flex;
+$room-h1-flex-direction: column;
+$room-h1-align-items: center;
+$room-h1-justify-content: center;
+$room-h1-position: absolute;
```

Comment:
You can delete from naming part room- . It's styles for all h1, not only for h1 on Room page.

Reply:
Done